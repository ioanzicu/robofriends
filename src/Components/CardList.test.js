import React from 'react'
import { shallow } from 'enzyme'
import CardList from './CardList'

it('expect to render CardList component', () => {
  const mockRobots = [
    {
      id: 1,
      name: 'Altor',
      username: 'Bino Altor',
      email: 'binoaltor@gmail.com'
    }
  ]
  const cardListComponent = shallow(<CardList robots={mockRobots} />)
  expect(cardListComponent.debug()).toMatchSnapshot()
})
