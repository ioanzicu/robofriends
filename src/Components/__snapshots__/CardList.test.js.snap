// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render CardList component 1`] = `
"<div>
  <Card id={1} name=\\"Altor\\" email=\\"binoaltor@gmail.com\\" />
</div>"
`;
